corehq/apps/app_manager/templates/app_manager/form_view_base.html

```diff
@@ -103,9 +103,18 @@
             var isModuleFilterEnabled = {{ is_module_filter_enabled|BOOL }};
             var formFilter = {{ form.form_filter|JSON }};
             var allOtherFormsRequireACase = {{ form.all_other_forms_require_a_case|BOOL }};
+            var formFilterReferencesCase = {{ form_filter_references_case|BOOL }};
             var formFilterAllowed = ko.computed(function () {
```

Comment:
I think I can answer the third one: The condition references a user case if `any((case in prepared_xpath for case in ('#user', UserCaseXPath().case())))`. I'm guessing it should be allowed if `domain.usercase_enabled` (or `is_usercase_in_use(domain_name)` which amounts to the same thing).